tools/data-model-exporter/data_model_exporter/dmExporter.py

```diff
@@ -128,6 +140,10 @@ def rdf_to_json(file_path, class_list):
     return json_schema_list
```

Comment:
consider rewriting this as a [dict comprehension](https://www.programiz.com/python-programming/dictionary-comprehension) to make it a bit more compact/readable